### PR TITLE
[Feature:Ducks!] seasonal ducks svg

### PR DESCRIPTION
**Pull request by hertacalvo**
Added seasonal ducks in svg format (January-July) at /site/public/img
File names are ##-monthname.svg format.

This PR changes no code and changes no existing files so testing is not necessary.

### CODE REVIEW

**Comment by hertacalvo** [comment]
Below are screenshots of each duck design (GitHub doesn't support svg here)

January
image

February
image

 March 
image

April 
image

May
image

June
image

July
image

**Comment by mukul-kmr-jha** [comment]
Wow!

File: site/app/controllers/GlobalController.php [hunk 1]
```diff
@@ -343,20 +343,75 @@ public function header() {
         }
 
         $now = getDate(date_timestamp_get($this->core->getDateTimeNow()));
-        $month = $now['mon'];
-        $day = $now['mday'];
+        $duck_img = $this->getDuckImage($now);
```
Comment (MasterOdin) [inline site/app/controllers/GlobalController.php:346]: Can we just pass `$this->core->getDateTimeNow()` to `$this->getDuckImage()` and then use `$day = (int) $now->format('j')` and `$month = (int) $now->format('n`)` inside of the function.

Comment (bmcutler) [reply site/app/controllers/GlobalController.php]: done

**Pull request merged**